swap/log.go

```diff
@@ -0,0 +1,104 @@
+package swap
+
+import (
+	log "github.com/ethereum/go-ethereum/log"
+)
+
+const (
+	// CallDepth is set to 1 in order to influence to reported line number of
+	// the log message with 1 skipped stack frame of calling l.Output()
+	CallDepth = 1
+	//DefaultAction is the default action filter for SwapLogs
+	DefaultAction string = "undefined"
+)
+
+// Logger wraps the ethereum logger with specific information for swap logging
```

Comment:
What is meant with `ethereum logger`

Reply:
the comment could be clearer in terms of the logger being wrapped specifically with the swap `action`

Reply:
Added more information as to what this Logger is used for and it's action field.